packages/nc-gui/composables/useViewFilters.ts

```diff
@@ -129,6 +129,7 @@ export function useViewFilters(
         if (!autoApply?.value) {
           filter.status = 'delete'
           // if auto-apply enabled invoke delete api and remove from array
+          filters.value.splice(i, 1)
```

Comment:
If `autoApply` is disabled we should just mark it as `deleted` only after applying should remove it from the array. In the above case, pressing the apply button filter won't get deleted since it's already removed from the array.


Uploading Screen Recording 2022-10-12 at 12.04.18 PM.mov…